src/components/HelpDialog.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Tooltip from '@material-ui/core/Tooltip';
+import HelpOutlineIcon from '@material-ui/icons/HelpOutline';
+import Dialog from "@material-ui/core/Dialog";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import Button from "@material-ui/core/Button";
+
+export default function HelpDialog(props) {
+    const [helpOpen, setHelpOpen] = React.useState(false);
+
+    return (<span>
+        <Tooltip title='Click for more info'>
+            <HelpOutlineIcon fontSize='small' onClick={() => setHelpOpen(true)}/>
```

Comment:
Is it possible to make this gray instead of black? It's a little more prominent than necessary imo. I'm not great at design though so feel free to fight me on it ;)

Reply:
using `color="action"` makes it this dark gray:
![image](https://user-images.githubusercontent.com/35227304/108140151-7fb2f180-708f-11eb-8536-8ba9d9d8e9a1.png)

`color="disabled"` is lighter:
![image](https://user-images.githubusercontent.com/35227304/108140224-a6712800-708f-11eb-8a10-462ece55b7dd.png)

can also do pink or purple (bc they're the primary/secondary colors), but I don't think we want that.

Reply:
I like it! Even though it's not technically disabled haha.

Reply:
currently I have it as `color="action"` (so the darker gray) - https://github.com/livgust/macovidvaccines.com/pull/21/files#diff-58bfc00b595a55aa3aef8861c670d301b372d97bfba315dd11556a247f145bbfR18

should I change it to disabled for the lighter color?